File: Said_Coley_Romero/Programas/Dart/Trabajo_19_de_Junio.dart
import 'dart:io';

void main(){
  print("Ingrese la cantidad de estudiantes:");
  int dattos = int.parse(stdin.readLineSync()!);

  for (int i = 1; i <= dattos; i++) {
    print("Diga su nombre acontinuacion:");
    String n4m3 = stdin.readLineSync()!;

    print("Diga su municipio acontinuacion:");
    String mUn1 = stdin.readLineSync()!;

    print("Diga su placa acontinuacion:");
    String pl4ka = stdin.readLineSync()!;

    var n4m3cantt = n4m3.substring(n4m3.length - 2);
    var mUn1cantt = mUn1.substring(0,3);
    var pl4kacantt = pl4ka.substring(pl4ka.length - 3);
  
  print('Su codigo es: ' + mUn1cantt + n4m3cantt + pl4kacantt);
  }


}

File: Said_Coley_Romero/Programas/Ruby/Trabajo_19_de_Junio.rb
puts "Ingrese la cantidad de estudiantes:"
dattos = gets.chomp.to_i

for i in 1..dattos

    puts "Diga su nombre acontinuacion:"
    n4m3 = gets.chomp
    puts "Diga su municipio acontinuacion:"
    mUn1 = gets.chomp
    puts "Diga su placa acontinuacion:"
    plak4 = gets.chomp

    n4m3cantt = n4m3[(n4m3.length-2)..n4m3.length]
    mUn1cantt = mUn1[0..2]
    plak4cantt = plak4[(plak4.length-3)..plak4.length]
    puts "Su codigo es: " + mUn1cantt + n4m3cantt + plak4cantt
end